docs/docs/snapshots.md

```diff
@@ -6,83 +6,123 @@ title: Database Snapshots
 
 ```console
 $date
-Thu  1 Aug 2024 09:49:30 BST
+Mon 28 Jul 2025 08:24:59 GMT
 
 $curl -s -I -L https://juno-snapshots.nethermind.io/files/mainnet/latest | gawk -v IGNORECASE=1 '/^Content-Length/ { printf "%.2f GB\n", $2/1024/1024/1024 }'
-172.47 GB
+186.23 GB
 
 $curl -s -I -L https://juno-snapshots.nethermind.io/files/sepolia/latest | gawk -v IGNORECASE=1 '/^Content-Length/ { printf "%.2f GB\n", $2/1024/1024/1024 }'
-5.67 GB
+30.45 GB
 
 $curl -s -I -L https://juno-snapshots.nethermind.io/files/sepolia-integration/latest | gawk -v IGNORECASE=1 '/^Content-Length/ { printf "%.2f GB\n", $2/1024/1024/1024 }'
-2.4 GB
+6.14 GB
 ```
 
 ## Run Juno with a snapshot
 
-### 1. Download the snapshot
+This method downloads and extracts the snapshot in one step without requiring double disk space:
+
+### 1. Prepare a directory
 
-First, download a snapshot from one of the provided URLs:
+Ensure you have a directory to store the snapshots. We will use the `$HOME/snapshots` directory:
 
 ```bash
-wget -O juno_mainnet.tar https://juno-snapshots.nethermind.io/files/mainnet/latest
+mkdir -p $HOME/snapshots
 ```
 
-### 2. Prepare a directory
+### 2. Install zstd
 
-Ensure you have a directory to store the snapshots. We will use the `$HOME/snapshots` directory:
+[zstd (Zstandard)](https://github.com/facebook/zstd) is required to decompress and directly stream the snapshots into your system without requiring temporary storage. zstd provides significantly better compression ratios and faster decompression speeds compared to traditional tar compression.
 
 ```bash
-mkdir -p $HOME/snapshots
+# On Ubuntu/Debian
+sudo apt-get install zstd
+
+# On macOS
+brew install zstd
+
+# On RHEL/CentOS/Fedora
+sudo dnf install zstd  # or yum install zstd
 ```
 
-### 3. Extract the snapshot
+### 3. Stream download and extract
 
-Extract the contents of the downloaded `.tar` file into the directory:
+Download and extract the snapshot directly to your target directory:
 
 ```bash
-tar -xvf juno_mainnet.tar -C $HOME/snapshots
+# For Mainnet
+curl -s -L https://juno-snapshots.nethermind.io/files/mainnet/latest \
+| zstd -d | tar -xvf - -C $HOME/snapshots
 ```
 
-### 4. Run Juno
+For other networks, replace the URL with:
+- **Sepolia**: `https://juno-snapshots.nethermind.io/files/sepolia/latest`
+- **Sepolia-Integration**: `https://juno-snapshots.nethermind.io/files/sepolia-integration/latest`
+
+## Alternative method: Download then extract
+
+If you prefer the traditional two-step approach or have limited bandwidth, you can download the snapshot first and extract it later:
+
+### 1. Download the snapshot
+
+```bash
+# For Mainnet
+wget -O juno_mainnet.tar.zst https://juno-snapshots.nethermind.io/files/mainnet/latest
+
+# Or using curl
+curl -L https://juno-snapshots.nethermind.io/files/mainnet/latest -o juno_mainnet.tar.zst
+```
+
+### 2. Extract the snapshot
+
+```bash
+# Extract to your snapshots directory
+zstd -d juno_mainnet.tar.zst -c | tar -xvf - -C $HOME/snapshots
+```
```

Comment:
Then all of this subsections should get an added level, baseed on what the above section `Alternative ...` changed